gui/src/components/onboarding/system-config/date-time/date-time-config.vue

```diff
@@ -43,7 +43,7 @@
                 v-model.trim="setDateTime.NtpServerAddress"
                 id="txtDTHostname"
                 @input="$v.setDateTime.NtpServerAddress.$touch"
-                placeholder="time.seagate.com"
+                placeholder="ntp-b.nist.gov"
```

Comment:
Place holder is just used and example. 
Making this change will get reflected in LDR product also which i think may not be acceptable and need an acceptance from PLM.

Reply:
May be having a generic text which explains NTP in few words with a sample may be better